### PR TITLE
CDAP-5803 ODBC driver documentation

### PR DESCRIPTION
**Pull request by awholegunch**
Running as [Quick Build 3](http://builds.cask.co/browse/CDAP-DOB197-3).
Passed [Quick Build 2](http://builds.cask.co/browse/CDAP-DOB197-2).
Pages of interest:

- [Integrations Index Page](http://builds.cask.co/artifact/CDAP-DOB197/shared/build-2/Docs-HTML/3.4.0-SNAPSHOT/en/integrations/index.html)
- [ODBC Page](http://builds.cask.co/artifact/CDAP-DOB197/shared/build-2/Docs-HTML/3.4.0-SNAPSHOT/en/integrations/odbc.html)

**Note**: A change was made in the CSS to improve the display of images; to keep a consistent API, the Ambari Installation and Hydrator files were updated as to the usage of the CSS. No other changes were made to those files.

### CODE REVIEW

File: cdap-docs/integrations/source/index.rst [hunk 1]
```diff
@@ -57,9 +57,9 @@ Integrations
 - |mapr|_ Configuring and installing CDAP on **MapR** *(Administration Manual)*
 
 
-.. rubric:: BI Tools: Accessing CDAP Datasets through Business Intelligence Tools
+.. rubric:: Accessing CDAP Datasets through JDBC, ODBC, and Business Intelligence Tools
```
Comment (sreevatsanraman) [inline cdap-docs/integrations/source/index.rst]: We can just say Accessing CDAP Datasets through JDBC and ODBC. BI tools do access them through that.

**Comment by sreevatsanraman** [comment]
Can we add a compatibility matrix? Which version of ODBC driver will work with what versions of CDAP. 
For now, it will hvae one row and one column and we should say the current ODBC driver will work with CDAP 3.4+

**Comment by sreevatsanraman** [comment]
LGTM otherwise

File: cdap-docs/integrations/source/index.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/integrations/source/index.rst]: Fixed.

**Comment by sreevatsanraman** [comment]
LGTM

**Pull request merged**